File: app/src/main/java/com/anokmik/tripassistant/databinding/BindingConverters.java
package com.anokmik.tripassistant.databinding;

import android.databinding.BindingConversion;
import android.databinding.ObservableBoolean;

public final class BindingConverters {

    private BindingConverters() {

    }

    @BindingConversion
    public static boolean convertObservableToBoolean(ObservableBoolean observableBoolean) {
        return observableBoolean.get();
    }

}

File: app/src/main/kotlin/com/anokmik/tripassistant/databinding/BindingConverters.kt
package com.anokmik.tripassistant.databinding

import android.databinding.BindingConversion
import android.databinding.ObservableBoolean

object BindingConverters {

    @JvmStatic
    @BindingConversion
    fun convertObservableToBoolean(observableBoolean: ObservableBoolean): Boolean {
        return observableBoolean.get()
    }

}